github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -797,7 +797,6 @@ presubmits:
         securityContext:
           privileged: true
   - always_run: false
-    run_before_merge: true
```

Comment:
This is needed for phased plugin. We will need to check why the automation is removing this. 

/hold

Reply:
we can create a patched struct that have the new field, wdyt ?